%YAML:1.0

# Bag Info
preRectified: 1
startTime: 8
bagPath: "/media/ygling2008/DATA/KITTI/stereo/03/"
lidarPath: "/media/ygling2008/DATA/KITTI/lidar/03/velodyne/"
gtPoseFile: "/media/ygling2008/DATA/KITTI/poses/03.txt"
imageTotalNum: 800
flipStereo: 0

# Debug Info
displayViewer: 1

# Mesh parameters
renderingK: 20
displayDelaunyResult: 1
displayCenterFrame: 1
displayUnOptimizedPoints: 0
meshSubdivNum: 0
pointQualityThreshold: 0
cvWaitTime: 1
pointUpdateThreshold: 1
constraintUpdateThreshold: 1.5
distanceThreshold: 0.3

#--------------------------------------------------------------------------------------------
# Camera Parameters. Adjust them!
#--------------------------------------------------------------------------------------------

# Camera calibration and distortion parameters (OpenCV) 
Camera.fx: 721.5377
Camera.fy: 721.5377
Camera.cx: 609.5593
Camera.cy: 172.854

Camera.k1: 0.0
Camera.k2: 0.0
Camera.p1: 0.0
Camera.p2: 0.0

# Camera frames per second 
Camera.fps: 10.0

# stereo baseline times fx
Camera.bf: 387.5744

# Color order of the images (0: BGR, 1: RGB. It is ignored if images are grayscale)
Camera.RGB: 1

# Close/Far threshold. Baseline times.
ThDepth: 40

#--------------------------------------------------------------------------------------------
# ORB Parameters
#--------------------------------------------------------------------------------------------

# ORB Extractor: Number of features per image
ORBextractor.nFeatures: 2000

# ORB Extractor: Scale factor between levels in the scale pyramid 	
ORBextractor.scaleFactor: 1.2

# ORB Extractor: Number of levels in the scale pyramid	
ORBextractor.nLevels: 8

# ORB Extractor: Fast threshold
# Image is divided in a grid. At each cell FAST are extracted imposing a minimum response.
# Firstly we impose iniThFAST. If no corners are detected we impose a lower value minThFAST
# You can lower these values if your images have low contrast			
ORBextractor.iniThFAST: 20
ORBextractor.minThFAST: 7

#--------------------------------------------------------------------------------------------
# Viewer Parameters
#--------------------------------------------------------------------------------------------
Viewer.KeyFrameSize: 0.05
Viewer.KeyFrameLineWidth: 1
Viewer.GraphLineWidth: 0.9
Viewer.MeshLineWidth: 0.8
Viewer.PointSize: 2
Viewer.CameraSize: 0.3
Viewer.CameraLineWidth: 3
Viewer.ViewpointX: 0
Viewer.ViewpointY: 0
Viewer.ViewpointZ: 1
Viewer.ViewpointF: 721.5377



